.github/workflows/dependabot-versioning-prs.yml

```diff
@@ -1,11 +1,12 @@
 # workflow to update the framework cli versioning PRs we get
 # from dependabot for C3
 
-name: "C3: Generate changeset for dependabot PRs"
+name: "Generate changesets for dependabot PRs"
 on:
   pull_request_target:
     paths:
       - "packages/create-cloudflare/src/frameworks/package.json"
+      - "packages/miniflare/package.json"
```

Comment:
I wonder if this is what we want, or if we actually want separate workflows for miniflare and c3 each with their respective triggers. I'm thinking about the case where one package has changes but the other doesn't.

It seems like `generate-dependabot-pr-changesets.ts` will tolerate being called on a target even if there are no changes (it will just throw a warning saying so)

Reply:
Hmm. Good point. Let's split them into two but keep the shared helper.